aws/resource_aws_sns_topic_subscription_test.go

```diff
@@ -749,7 +749,6 @@ resource "aws_api_gateway_authorizer" "test" {
   name                             = "tf-acc-test-api-gw-authorizer-%d"
   rest_api_id                      = "${aws_api_gateway_rest_api.test.id}"
   authorizer_uri                   = "${aws_lambda_function.authorizer.invoke_arn}"
-  authorizer_result_ttl_in_seconds = "0"
```

Comment:
this fix looks good! I also noticed in this test step https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/resource_aws_api_gateway_authorizer_test.go#L190 we're able to update the TTL to 0 without a problem so it looks like the error here is attributed to the `d.GetOk` call instead of `d.Get` when setting this value at creation time https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/resource_aws_api_gateway_authorizer.go#L111 . might be out of scope to address it in this PR but I can make in issue/see if there already is one stating we can't create this authorizer resource w/ a value of 0 (caching disabled)

Reply:
Found: #12633